app/views/content/non-uk-teachers/teach-in-england-if-you-trained-overseas.md

```diff
@@ -8,7 +8,7 @@ related_content:
     A day in the life of a teacher in England : "/a-day-in-the-life-of-a-teacher"
 promo_content:
 navigation: 20.20
-navigation_title: Teach in England if you trained outside the UK
+navigation_title: Teach in England as a non-UK qualified teacher
```

Comment:
Why are we changing the nav? Might this be better as part of a wider piece of work?

Reply:
I think the new version is clearer – do you not agree?